lib/workers/qemu.js

```diff
@@ -16,12 +16,21 @@
 
 'use strict'
 
+const _ = require('lodash')
 const childProcess = require('child_process')
 
-const childKillEventHandler = (child, event) => {
-  process.on(event, (code) => {
+const manageHandlers = (child, register) => {
+  const action = register ? 'once' : 'removeListener'
+
+  process[action]('exit', (code) => {
     child.kill()
-    process.exit(code || 0)
+    process.exitCode = code || 0
```

Comment:
Very low priority, but I think you can instead do:

```js
process[action]('exit', (code = 0) => {
    child.kill()
    process.exitCode = code
})
```

Which looks a bit nicer :)